ui/src/dashboards/actions/index.js

```diff
@@ -89,10 +106,9 @@ export const deleteDashboardCell = (cell) => ({
 
 // Async Action Creators
 
-export const getDashboards = (dashboardID) => (dispatch) => {
-  getDashboardsAJAX().then(({data: {dashboards}}) => {
-    dispatch(loadDashboards(dashboards, dashboardID))
-  })
+export const getDashboardsAsync = (dashboardID) => async (dispatch) => {
+  const {data: {dashboards}} = await getDashboardsAJAX()
```

Comment:
It's good you're moving legacy code to async/await, but should this await be wrapped in a try/catch? Most async/awaits should probably be, considering when they fail, that failure is often silent. I know there's probably nothing more we can do but log the error, but that helps with debugging.